prisma/generated/schemas/aggregateCompany.schema.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from 'zod'
```

Comment:
We should likely add `prisma/generated` to .gitignore.

Reply:
👌🏻 Add a script to the docker file and ignore the generated folder?

Reply:
`RUN npx prisma generate` before cmd?